frontend/src/app/core/functions/get-role.ts

```diff
@@ -0,0 +1,8 @@
+export function getRole(): string {
```

Comment:
where is it set? where is it used? How can you tell the role of user based on random token?

Reply:
it is set after logging in. We send request to Get user date endpoint and save it